apollo-training/src/modules/schema.graphql

```diff
@@ -1,3 +1,11 @@
 type Query {
   getMyProfile: User!
+  getAllTrainees: [User]
+  getTrainee(id: ID!): User!
```

Comment:
Why (!) is used and What is ID type

Reply:
(!) exclamation mark is just use for important and required.

Reply:
ID type ?